docs/content/guides/observability/ui/explore.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Explore the Gloo UI
+weight: 20
+description: Explore the features of the Gloo UI to help you get an at-a-glance view of your Gloo Gateway environment.
+---
+
+Explore the features of the Gloo UI to help you get an at-a-glance view of your Gloo Gateway environment.
+
+## About the Gloo UI
+
+* **Gloo Gateway dashboard**: Use the Gloo UI dashboard to quickly review the health of your Gloo Gateway setup, and any insights that were detected. If you use the Gloo UI in a multicluster setup, you can also review the health of the Gloo management server and agents. 
+* **Insights**: The Gloo UI comes with an insights engine that automatically analyzes your Gloo Gateway setup for health issues. These issues are shared in the UI along with recommendations to harden your Gloo Gateway setup. The insights give you a checklist to address issues that might otherwise be hard to detect across your environment. For more information, see [Insights](#insights).
+* **Traffic resource overview**: Review the Gateways, routes, policies, and destinations that are set up in your environment and the traffic management rules that you applied to them. 
```

Comment:
maybe lowercase? unless you mean the Gateway CR and not such for the for other things
```suggestion
* **Traffic resource overview**: Review the gateways, routes, policies, and destinations that are set up in your environment and the traffic management rules that you applied to them. 
```